pkg/sidecar/sidecar.go

```diff
@@ -53,53 +51,59 @@ func Run(runnerName string) error {
 
 	defer manager.Close()
 
-	return manager.Manage(ctx, func(ctx context.Context, instance *Instance) error {
-		instance.S().Infow("managing instance", "instance", instance.Hostname)
+	return manager.Manage(globalctx, func(ctx context.Context, instance *Instance) error {
+		instance.S().Debugw("managing instance", "instance", instance.Hostname)
 
 		defer func() {
+			instance.S().Debugw("closing instance", "instance", instance.Hostname)
 			if err := instance.Close(); err != nil {
 				instance.S().Warnf("failed to close instance: %s", err)
 			}
 		}()
 
-		g, ctx := errgroup.WithContext(ctx)
-
 		// Network configuration loop.
-		g.Go(func() error {
```

Comment:
Did we only ever schedule one goroutine in this errgroup?

Reply:
They used to be two, but since it is only one now, I am removing the error group.